framework/web/widgets/CHtmlPurifier.php

```diff
@@ -110,7 +110,7 @@ public function getOptions()
 	 * Get the HTML Purifier instance or create a new one if it doesn't exist.
 	 * @return HTMLPurifier
 	 */
```

Comment:
I agree that the framwork should not be changed. Can you properly extend CHTMLPurifier instead, please?

Reply:
I extended CHtmlPurifier by creating a helper class. However, I don't know how LimeSurvey usually deals with these problems, so feel free to tell me another way if this doesn't comply.

Reply:
@Shnoulle Any idea?

Reply:
I don't know, in general we extend Yii class in `core` if i don't make error. 

But i have same issue for a project in develop : in helpers it's more easy because you can use autoloader. I don't have fixed idea of the good place.

For example : https://github.com/LimeSurvey/LimeSurvey/blob/master/application/core/LSETwigViewRenderer.php or https://github.com/LimeSurvey/LimeSurvey/blob/ba08ebf863baa4dd7b770093c7e1904b7325b92a/application/core/LSCGettextMessageSource.php#L10

`class LSYiiClassName extends YiiClassName` maybe